pkg/bugzilla/bugzilla.go

```diff
@@ -36,9 +36,8 @@ func NewVerifier(bzClient bugzilla.Client, ghClient github.Client, pluginConfig
 	}
 }
 
-// pr contains a bugzilla bug ID and the associated GitHub pr that resolves the bug
-type pr struct {
-	bugID int
+// PR contains the org, repo, and PR number for a PR
+type PR struct {
```

Comment:
Exposing a struct but none of it's fields is weird - does this thing have exported methods?

Reply:
No, it doesn't need to be exported. Changed it back.